docs/man-pages/condor_hold.rst

```diff
@@ -1,114 +1,122 @@
-      
-
 *condor_hold*
-==============
+=============
+
+Place job(s) into the hold state in HTCondor.
 
-put jobs in the queue into the hold state
-:index:`condor_hold<single: condor_hold; HTCondor commands>`\ :index:`condor_hold command`
+:index:`condor_hold<double: condor_hold; HTCondor commands>`
 
 Synopsis
 --------
 
-**condor_hold** [**-help | -version** ]
+**condor_hold** [**-help** | **-version**]
 
-**condor_hold** [**-debug** ] [**-reason** *reasonstring*]
-[**-subcode** *number*] [
-**-pool** *centralmanagerhostname[:portnumber]* |
-**-name** *scheddname* ] | [**-addr** *"<a.b.c.d:port>"*]
-*cluster... | cluster.process... | user...* |
-**-constraint** *expression* ...
+**condor_hold** [*OPTIONS*] [*cluster*... | *cluster.proc*... | *user*...]
 
-**condor_hold** [**-debug** ] [**-reason** *reasonstring*]
-[**-subcode** *number*] [
-**-pool** *centralmanagerhostname[:portnumber]* |
-**-name** *scheddname* ] | [**-addr** *"<a.b.c.d:port>"*] **-all**
+**condor_hold** [**-debug**] [**-long**] [**-totals**] [**-all**]
+[**-constraint** *expression*] [**-reason** *message*] [**-subcode** *number*]
+[**-pool** *hostname[:portnumber]* | **-name** *scheddname* | **-addr** *"<a.b.c.d:port>"*]
 
 Description
 -----------
 
-*condor_hold* places jobs from the HTCondor job queue in the hold
-state. If the **-name** option is specified, the named *condor_schedd*
-is targeted for processing. Otherwise, the local *condor_schedd* is
-targeted. The jobs to be held are identified by one or more job
-identifiers, as described below. For any given job, only the owner of
-the job or one of the queue super users (defined by the
-:macro:`QUEUE_SUPER_USERS` macro) can place the job on hold.
-
-A job in the hold state remains in the job queue, but the job will not
-run until released with *condor_release*.
-
-A currently running job that is placed in the hold state by
-*condor_hold* is sent a hard kill signal.
+Place job(s) into the hold state in HTCondor. Held jobs will remain
+in HTCondor but will not be matched to resources or executed. For
+any given job, only the owner of the job or one of the queue super
+users (defined by the :macro:`QUEUE_SUPER_USERS` macro) can place
+the job on hold.
```

Comment:
Instead of redefining queue super users here, can we just leave the link, e..g something like

"... one of the :macro:`QUEUE_SUPER_USERS` can hold the job"

Also, let's try to use the verb "hold" in this man page instead of "place the job on hold".